source/adressering-av-undertegner.rst

```diff
@@ -1,78 +1,76 @@
+    How you choose to address the signer(s) affects the :ref:`content and appearance of the signature(s) in the resulting signed document <identify-signers>`.
 
-1. Adressering på e-post / SMS
+1. Addressing signers by e-mail / SMS
 ===============================
 
 .. NOTE::
-   Dette alternativet er kun tilgjengelig for private virksomheter
+   This option is only available for private organizations
 
-Ved adressering på e-post / SMS knyttes ikke signeringsforespørselen til et fødselsnummer. Vi har derfor ikke noen måte å verifisere at det er riktig person som leser og signerer. Det kreves derfor ikke innlogging for å lese og signere. Mottaker får e-post og/eller SMS med en unik kode og lenke som gir tilgang til forespørselen. Se :ref:`varsler <varslerUtenFødselsnummer>` for å se hvordan varselet ser ut.
+On addressing by email/SMS, the signature request is not linked to a national identity number. We therefore do not have any way of verifying that the correct person is reading and signing. It is therefore not necessary to log in so as to read and sign. The recipient receives an email and/or SMS with a unique code and link to access the request. View :ref:`notifications <notifications-without-national-identity>` to see what the notification looks like.
 
 
 ..  IMPORTANT::
-    Selve signaturen er like sikker og gyldig som når du adresserer med fødselsnummer, men du som avsender er ansvarlig for at riktig person leser og signerer.
+    The signature itself is just as secure and valid as when you address using national identity number, but you as the sender are responsible for ensuring that the correct person reads and signs.
 
-Ved adressering på e-post / SMS blir ikke fødselsnummeret bli inkludert i det signerte dokumentet, av personvernmessige hensyn. Du vil fortsatt få med navn og eventuelt fødselsdato i det signerte dokumentet.
+In the case of e-mail/SMS addressing, the national identity number will not be included in the signed document for privacy reasons. Name and any date of birth will continue to be included in the signed document.
 
-Se :ref:`Signering i portalflyt med adressering på e-post / SMS <signering-i-portalflyt-uten-fødselsnummer>` for mer informasjon og visuelle eksempler.
+See :ref:`Signing in portal flow with addressing by e-mail / SMS <signing-in-portal-flow-without-national-identity>` for more information and visual examples.
 
-2. Adressering med fødselsnummer
+2. Address using national identity number
 ================================
-For å adressere med fødselsnummer må du vite undertegners fødselsnummer. Adressering med fødselsnummer er den sikreste måten å nå mottaker på siden det krever innlogging med elektronisk ID for å lese og signere.
+To address using a national identity number, you must know the signer's national identity number. Addressing using a national identity number is the safest way to reach the recipient, since it requires logging in with an electronic ID in order to read and sign.
```

Comment:
Snur om litt fordi setningen virket litt sånn åpenbar slik den var skrevet fra før (for å bruke noe informasjon må du vite informasjonen liksom.)

```suggestion
If you know the national identity numbers of the signers, you can address them using their identity numbers. This is the most secure way to reach the expected recipients, since it requires them to log in with an electronic ID in order to read and sign.
```